.travis.yml

```diff
@@ -8,8 +8,8 @@ matrix:
       env: COMPILER=4.03
     - os: linux
       env: COMPILER=4.04
-    - os: linux
-      env: COMPILER=4.05
+    #- os: linux
+    #  env: COMPILER=4.05
```

Comment:
I think the bug may actually be related to a problem with 4.05+beta3, and it will work with 4.05+trunk